telegram/chatmember.py

```diff
@@ -123,7 +123,8 @@ def __init__(self, user, status, until_date=None, can_be_edited=None,
                  can_restrict_members=None, can_pin_messages=None,
                  can_promote_members=None, can_send_messages=None,
                  can_send_media_messages=None, can_send_polls=None, can_send_other_messages=None,
-                 can_add_web_page_previews=None, is_member=None, custom_title=None, **kwargs):
+                 can_add_web_page_previews=None, is_member=None, custom_title=None,
+                 **kwargs):
```

Comment:
why does kwargs get its own line here? Is that a general custom we follow? If yes, why in this PR? You just saw it?